tests/logging_tests/tests.py

```diff
@@ -109,16 +110,82 @@ def test_django_logger_debug(self):
         self.assertEqual(self.logger_output.getvalue(), '')
 
 
+class LoggingAssertionMixin(object):
+    def assert_request_logs(self, url, level, msg, status_code,
```

Comment:
We usually model these names off of the unittest came case convention, e.g. assertRequestLogs